maildir/maildir_private.h

```diff
@@ -96,7 +96,7 @@ int                     maildir_check_dir      (struct Mailbox *m, const char *d
 void                    maildir_delayed_parsing(struct Mailbox *m, struct Maildir **md, struct Progress *progress);
 struct MaildirMboxData *maildir_mdata_get      (struct Mailbox *m);
 int                     maildir_mh_open_message(struct Mailbox *m, struct Message *msg, int msgno, bool is_maildir);
-int                     maildir_move_to_context(struct Context *ctx, struct Maildir **md);
+int                     maildir_move_to_context(struct Mailbox *m, struct Maildir **md);
 int                     maildir_parse_dir      (struct Mailbox *m, struct Maildir ***last, const char *subdir, int *count, struct Progress *progress);
 void                    maildir_parse_flags    (struct Email *e, const char *path);
```

Comment:
Not an issue, but this one out of alignment parenthesis really bothers me

Reply:
I like your attention to detail, but I can't see any mis-alignment.

Reply:
I must have been delirious because I could have sworn a parenthesis was one column to the left of the others